### PR TITLE
Symmetry mode UI

### PR DESCRIPTION
**Pull request by torcado194**
Adds UI for selecting tile symmetry options

Couple notes:
- I modified `TerrainUndo.create_peering_restore_point_tile` to include tile symmetry, so that e.g. copy/paste operations handled symmetry properly. It doesn't necessarily fit the function name anymore, so if you think it should be renamed (or be made a separate function) I can change that
- I hijacked this PR to add a small QOL feature I've wanted, toggling the terrain type of selected tiles when pressing `enter`

Thanks again for the initial work on this feature :)

### CODE REVIEW

**Comment by torcado194** [comment]
whoops i selected the wrong branch

**Comment by Portponky** [comment]
Couple issues

* When all paint modes are unselected, you can still paint stuff (e.g. peering bits or rotations)
* When all paint modes are unselected, you still see the last thing selectd. It should probably show peering bits in this case.
* You can edit symmetry settings for terrain types you don't have selected (and thus can't see the symmetry of).

Looks great otherwise.

**Comment by Portponky** [comment]
The tooltip for the symmetry mode button needs to be updated too.

**Comment by torcado194** [comment]
done! 👍 

It looks like `match` doesn't match on `null`? I couldn't find any documentation about it from a quick search, I changed it to the default selector `_` which fixes the unselected button issues

File: addons/better-terrain/editor/Dock.gd [hunk 1]
```diff
@@ -101,6 +104,13 @@ func _ready() -> void:
 	tile_view.paste_occurred.connect(_on_paste_occurred)
 	tile_view.change_zoom_level.connect(_on_change_zoom_level)
 	tile_view.terrain_updated.connect(_on_terrain_updated)
+	
+	symmetry_options.select(0)
+	symmetry_options.item_selected.connect(_on_symmetry_selected)
```
Comment (Portponky) [inline addons/better-terrain/editor/Dock.gd:109]: This is already connected in the editor/scene, so it's not needed here.

Comment (torcado194) [reply addons/better-terrain/editor/Dock.gd]: 👍 
this was actually a consequence of `@tool` mode updating those properties in-place and then me subsequently pushing those changes.
same goes for `.visible = false`, unless I'm mistaken

**Comment by Portponky** [comment]
Looks good, merging back to the symmetry branch. I'll try to get that merged back soon.

**Pull request merged**